cmd/edenConfig.go

```diff
@@ -55,6 +56,19 @@ func reloadConfigDetails() {
 		qemuFileToSave = utils.ResolveAbsPath(viper.GetString("eve.qemu-config"))
 		devModel = viper.GetString("eve.devmodel")
 		eveRemote = viper.GetBool("eve.remote")
+		modelFile = viper.GetString("eve.devmodelfile")
+		if modelFile != "" {
+			filePath, err := filepath.Abs(modelFile)
+			if err != nil {
+				log.Errorf("cannot get absolute path for devmodelfile (%s): %v", modelFile, err)
```

Comment:
Shouldn't this error be more than a log? log.Fatalf like the other issues with arguments?